content/contributing-terms.en.md

```diff
@@ -75,20 +75,24 @@ Within the `documents` JSON object, we will now declare terms.
 
 ## Declaring terms
 
-Terms are declared in a service declaration file, under the `documents` property. The way in which each type of terms should be obtained is declared as a JSON object.
+Terms are declared in a service declaration file, under the `documents` property. 
+
+Most of the time, terms are constituted by only one online source document (for example [Facebook Terms of Service](https://www.facebook.com/legal/terms)) but sometimes terms can be constituted by multiple online source documents (for example [Facebook community guidelines](https://transparency.fb.com/fr-fr/policies/community-standards/)) and it's all theses source documents combined together that finally constitute the terms. 
+
+First of all, let's see what constitute a source document.
```

Comment:
I am not sure of the added value of this sentence, and I believe it is slightly misleading as we don't define in the next paragraph _what constitutes_ a source document but rather _how it is represented_ in Open Terms Archive.

```suggestion
```

Reply:
I agree.